packages/hardware-trezor/src/transformers/withdrawals.ts

```diff
@@ -0,0 +1,42 @@
+import * as Trezor from 'trezor-connect';
+import { Cardano } from '@cardano-sdk/core';
+import { InvalidArgumentError } from '@cardano-sdk/util';
+import { TrezorTxTransformerContext } from '../types';
+import { resolveStakeKeyPath } from './keyPaths';
+
+export const toWithdrawal = (
+  withdrawal: Cardano.Withdrawal,
+  context: TrezorTxTransformerContext
+): Trezor.CardanoWithdrawal => {
+  const address = Cardano.Address.fromString(withdrawal.stakeAddress);
+  const rewardAddress = address?.asReward();
+
+  if (!rewardAddress) throw new InvalidArgumentError('withdrawal', 'Invalid withdrawal stake address');
+
+  let trezorWithdrawal;
+  if (rewardAddress.getPaymentCredential().type === Cardano.CredentialType.KeyHash) {
```

Comment:
Let's add a comment here to explain what this means in context of withdrawals

Reply:
oke we can do that. I thought it is clear enough :(

Reply:
✅

Reply:
Thanks for the clarification 🙏